core/dbt/adapters/sql/impl.py

```diff
@@ -182,7 +184,7 @@ def drop_schema(self, database, schema):
         self.execute_macro(DROP_SCHEMA_MACRO_NAME,
                            kwargs=kwargs)
 
-    def list_relations_without_caching(self, information_schema, schema):
+    def list_relations_without_caching(self, information_schema, schema) -> List[BaseRelation]:
```

Comment:
This line is too long and will end up failing pylint, it's too many characters! There are a other more lines like this too (119, 81, 77, 72)
```suggestion
    def list_relations_without_caching(
        self, information_schema, schema
    ) -> List[BaseRelation]:
```

Reply:
Thanks, I've set the line length to 80.